verifiers/utils/builder.go

```diff
@@ -9,6 +9,25 @@ import (
 	serrors "github.com/slsa-framework/slsa-verifier/v2/errors"
 )
 
+// TrustedAttesterID represents an identifier that has been explicitly trusted.
+// TODO: don't embed TrustedBuilderID, use a whole new type that implements the
```

Comment:
As someone not familiar with the codebase, why is this?

Reply:
I decided to remove it. It seems like it was originally for semver-like capabilities. Instead, I'll just refer the user in comments to a another semver package.
 - https://github.com/slsa-framework/slsa-verifier/pull/256
 - https://pkg.go.dev/golang.org/x/mod/semver